libminifi/src/core/Processor.cpp

```diff
@@ -101,14 +102,27 @@ void Processor::setScheduledState(ScheduledState state) {
 }
 
 bool Processor::addConnection(std::shared_ptr<Connectable> conn) {
-  bool ret = false;
+  enum class DidSet{
```

Comment:
Nitpicking: maybe "SetAs" would be a better name to represent what's going on here.

Reply:
interesting because I recall that I first called it "SetAs" but the commit history doesn't confirm this, will make the change

Reply:
done